<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProdutoItem">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>129</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsere">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>159</y>
      <width>711</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>cod.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>item</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>quant.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>preco</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>subtot</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>total</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="ldquantidade">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>129</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CAIXA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ld_codProduto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>129</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>109</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cod. Prod.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>220</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Descrição do Produto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Quant.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tipo Oper.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btngerar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Imprimir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dtEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProdutoItem_2">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>80</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ld_codProduto_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cod. Cliente.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dtEdit_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>59</y>
      <width>220</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nome do Cliente</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
